pkg/storage/couchdb/couchdbstore.go

```diff
@@ -26,6 +26,9 @@ import (
 
 const (
 	logModuleName = "edge-core-couchdbstore"
+
+	designDocumentFilteredOutLogMsg = "Currently getting all documents from a CouchDB store. " +
+		"A document with id %s was filtered out since it appears to be a CouchDB design document."
```

Comment:
I would change a bit the message here - make it shorter and remove uncertainty: "~Currently~ getting all documents from a CouchDB store. A document with id %s was filtered out ~since it appears to be~ as it's a CouchDB design document."

Reply:
Done!